diff -up purpose-5.67.0/src/externalprocess/purposeprocess_main.cpp.omv~ purpose-5.67.0/src/externalprocess/purposeprocess_main.cpp
--- purpose-5.67.0/src/externalprocess/purposeprocess_main.cpp.omv~	2020-02-21 04:28:13.267347979 +0100
+++ purpose-5.67.0/src/externalprocess/purposeprocess_main.cpp	2020-02-21 04:28:29.821348709 +0100
@@ -90,7 +90,7 @@ public:
 
 private Q_SLOTS:
     void error() {
-#if (QT_VERSION < QT_VERSION_CHECK(5, 15, 0))
+#if (QT_VERSION < QT_VERSION_CHECK(5, 16, 0))
         const auto socketError = m_socket.error();
 #else
         const auto socketError = m_socket.socketError();
